sdks/python/apache_beam/transforms/trigger.py

```diff
@@ -1036,14 +1074,17 @@ class BatchGlobalTriggerDriver(TriggerDriver):
       index=0,
       nonspeculative_index=0)
 
-  def process_elements(self, state, windowed_values, unused_output_watermark):
+  def process_elements(self, state, windowed_values,
+                       unused_output_watermark=MIN_TIMESTAMP,
```

Comment:
Let's not provide default values here, as it's unclear what they should be.

Reply:
Ack, I set these to None.

Reply:
I meant not provide default (None or otherwise).

Reply:
Ah gotcha. I removed the default value on the output_watermarks, but the input_watermark needs a default value to keep backwards compatibility.